lib/python/milc.py

```diff
@@ -111,10 +111,11 @@ def format(self, record):
         return super(ANSIEmojiLoglevelFormatter, self).format(record)
 
 
-class ANSIStrippingFormatter(ANSIFormatter):
+class ANSIStrippingFormatter(ANSIEmojiLoglevelFormatter):
     """A log formatter that strips ANSI.
     """
     def format(self, record):
+        record.levelname = ansi_escape.sub('', record.levelname)
```

Comment:
```suggestion
        if UNICODE_SUPPORT:
            record.levelname = EMOJI_LOGLEVELS[record.levelname]
```

Reply:
This is done by the parent class, ANSIEmojiLoglevelFormatter.

Reply:
I forgot to change the parent class...